<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModelForm</class>
 <widget class="QWidget" name="ModelForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>734</width>
    <height>497</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="4" column="2">
      <widget class="QCheckBox" name="m_inClassifierCheck">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Наличие в классификаторе</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLineEdit" name="m_nameEdit"/>
     </item>
     <item row="2" column="2">
      <widget class="QCheckBox" name="m_addToAlbumCheck">
       <property name="text">
        <string>Добавить в альбом</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QCheckBox" name="m_isBugCheck">
       <property name="text">
        <string>Кривая модель</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Войска</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>3ds модель</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Текстура</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Название</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QComboBox" name="m_armyBox"/>
     </item>
     <item row="8" column="2">
      <widget class="QLabel" name="m_imageLabel">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="8" column="3">
      <widget class="QPushButton" name="m_refreshImageButton">
       <property name="text">
        <string>Обновить картинку</string>
       </property>
      </widget>
     </item>
     <item row="7" column="3">
      <widget class="QPushButton" name="m_loadTextureButton">
       <property name="text">
        <string>Загрузить текстуру</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Картинка</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>max модель</string>
       </property>
      </widget>
     </item>
     <item row="6" column="2">
      <widget class="QLabel" name="m_maxLabel">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="7" column="2">
      <widget class="QLabel" name="m_textureLabel">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="6" column="3">
      <widget class="QPushButton" name="m_loadMaxButton">
       <property name="text">
        <string>Загрузить max</string>
       </property>
      </widget>
     </item>
     <item row="5" column="3">
      <widget class="QPushButton" name="m_load3dsButton">
       <property name="text">
        <string>Загрузить 3ds</string>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QLabel" name="m_3dsLabel">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="9" column="3">
      <widget class="QPushButton" name="m_refresh3DButton">
       <property name="text">
        <string>Обновить в 3D
просмотре</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
